mlflow/tracking/fluent.py

```diff
@@ -235,11 +254,19 @@ def start_run(
     .. code-block:: text
         :caption: Output
 
-        parent run_id: 5ec0e7ae18f54c2694ffb48c2fccf25c
-        child run_id : 78b3b0d264b44cd29e8dc389749bb4be
+        parent run:
+        run_id: 8979459433a24a52ab3be87a229a9cdf
+        description: starting a parent for experiment 7
+        version tag value: v1
+        priority tag value: P1
+        --
+        child run:
+        run_id : 7d175204675e40328e46d9a6a5a7ee6a
+        --
+        Child runs of parent run_id: 8979459433a24a52ab3be87a229a9cdf
```

Comment:
Do we need this? The parent-run ID is printed in line 258.

Reply:
`print("child runs of parent run")` doesn't print out `print("child runs of parent run")`. Can we fix?